src/Microsoft.ML.Data/TrainCatalog.cs

```diff
@@ -256,6 +256,14 @@ internal BinaryClassificationTrainers(BinaryClassificationCatalog catalog)
                 Evaluate(x.Scores, labelColumnName), x.Scores, x.Fold)).ToArray();
         }
 
+        public BinaryPredictionTransformer<TModel> ChangeModelThreshold<TModel>(BinaryPredictionTransformer<TModel> model, float threshold)
```

Comment:
We should put XML comments on all public members.

Reply:
https://media.giphy.com/media/xT5LMzIK1AdZJ4cYW4/giphy.gif